jetty-core/jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -2068,6 +2070,35 @@ private ConfigurationParser(Pool.Entry<ConfigurationParser> entry)
             addCatalog(URI.create(catalogUrl.toExternalForm()));
         }
 
+        protected SAXParserFactory newSAXParserFactory()
+        {
+            // Use JVM default implementation (not the one found in the classloader from non-JVM sources)
+            SAXParserFactory factory = SAXParserFactory.newDefaultInstance();
```

Comment:
Can you explain how this is different from :
```suggestion
            // Use JVM default implementation (not the one found in the classloader from non-JVM sources)
            SAXParserFactory factory = super.newSAXParserFactory();
```

Reply:
super.newSAXParserFactory uses `SAXParserFactory.newInstance()` which does a lookup for XML Factories across the entire classloader.  Even oddball xerces implementations or alternative embedded implementations from other uber jars.

The `SAXParserFactory.newDefaultInstance() `doesn't do the lookup, and only uses what is in the JVM.